hack/verify-staging-client-go.sh

```diff
@@ -72,3 +72,16 @@ godep restore ${V} 2>&1 | sed 's/^/  /'
 
 echo "Testing staging/copy.sh"
 staging/copy.sh -d 2>&1 | sed 's/^/  /'
+
+# Check that the apimachinery Godeps revision is actually in Github
+echo "Checking that a valid k8s.io/apimachinery is reference in client-go"
```

Comment:
The idea: `hack/update-client-go.sh -s`.

This gives you a client which has not valid apimachinery upstream rev, but works great for WIP PRs.